pyproject.toml

```
@@ -0,0 +1,17 @@
+[build-system]
+requires = ["setuptools", "wheel"]
```

```suggestion
requires = ["setuptools>=61.0"]
```
ref: https://packaging.python.org/en/latest/tutorials/packaging-projects/#choosing-a-build-backend